iron-overlay-behavior.html

```diff
@@ -313,23 +317,19 @@
       if (this.opened) {
         this._prepareRenderOpened();
       }
-      if (this._openChangedAsync) {
-        this.cancelAsync(this._openChangedAsync);
+
+      // requestAnimationFrame for non-blocking rendering
+      if (this.__openChangedAsync) {
```

Comment:
It might be good to call this `__openChangedRaf` since it's not generated by `this.async`..